Documentation/grpc-metadata.md

```diff
@@ -168,7 +168,7 @@ Server side metadata sending and receiving examples are available [here](../exam
 
 ### Receiving metadata
 
-To read metadata sent by the client, the server needs to retrieve it from RPC context.
+To read metadata sent by the client, the server needs to retrieve it from RPC context using `FromIncomingContext`. Do not use `FromOutgoingContext` on server as it is for client-side only.
```

Comment:
- Please ensure that lines are wrapped at 80 cols.

- Also, make `FromIncomingContext` a hyperlink by pointing it to our godocs.

How about moving the last sentence to the "Sending metadata" section. Also, be doubly explicit there by saying something like:
Do not use `FromOutgoingContext` on the server to write metadata to be sent to the client. `FromOutgoingContext` is for client-side use only. 
(And here also, make them hyperlinks to our godocs).

Thanks.

Reply:
Done

Reply:
I think using reference style links would make the text more readable. Please see: https://www.markdownguide.org/basic-syntax/#reference-style-links